src/data/markdown/docs/02 javascript api/07 k6-experimental/04 tracing.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "tracing"
+excerpt: "k6 Tracing experimental API"
+---
+
+<ExperimentalBlockquote />
+
+
+This experimental module instruments k6 scripts HTTP calls with tracing information. 
+
+
+| Class/Function                                                           | Description                                                                                                               |
+| :----------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------ |
+| [instrumentHTTP](/javascript-api/k6-experimental/tracing/instrumenthttp) | instruments the k6 http module with tracing capabilities.                                                                 |
+| [Client](/javascript-api/k6-experimental/tracing/client)                 | configurable Client that exposes instrumented HTTP operations and allows selectively instrumenting requests with tracing. |
+| [Options](/javascript-api/k6-experimental/tracing/options)               | Options allows to configure the tracing instrumentation behavior.                                                         |
+
+## Example
+
+This example demonstrates how to use the tracing API to instrument every HTTP request made in a script with tracing information.
+
+<CodeGroup labels={["example-tracing-module.js"]} lineNumbers={[]} showCopyButton={[true]}>
+
+```javascript
+import http from "k6/http";
+import { check } from "k6";
+import tracing from "k6/experimental/tracing";
+
+// instrumentHTTP will ensure that all requests made by the http module
+// from this point forward will have a trace context attached.
+//
+// The first argument is a configuration object that
+// can be used to configure the tracer.
+//
+// Currently supported HTTP methods are: get, post, put, patch, head,
+// del, options, and request.
+tracing.instrumentHTTP({
+propagator: "w3c",
+});
+
+export default () => {
+ // the instrumentHTTP call in the init context replaced
+ // the http module with a version that will automatically
+ // attach a trace context to every request.
+ //
+ // Because the instrumentHTTP call was configured with the
+ // w3c trace propagator, this request will as a result have
+ // a `traceparent` header attached. 
+ const res = http.get("http://httpbin.org/get", {
+ headers: {
+ "X-Example-Header": "instrumented/get",
```

Comment:
```suggestion
   "X-Example-Header": "instrumented/get",
```

The below `"status is 200"` line should also be further indented. Though indentation is off in general in all examples here.